### PR TITLE
Update references to faucets in the docs

### PR DESCRIPTION
**Pull request by bradford-m**
DO NOT MERGE YET - faucets.chain.link not yet live

### CODE REVIEW

File: docs/Developer Reference/link-token-contracts.md [hunk 1]
```diff
@@ -92,7 +90,7 @@ LINK is native to Ethereum, so to use LINK on other chains, it must be bridged.
 
 > 🚰BSC Faucet
 >
-> Testnet LINK is available from <a href="https://linkfaucet.protofire.io/bsctest" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
+> Testnet LINK is available from <a href="https://faucets.chain.link/chapel" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
```
Comment (ainsleybc) [inline docs/Developer Reference/link-token-contracts.md:93]: I don't think these are community faucets any more - maybe the text should match what we say above for kovan/rinkeby?

File: docs/Developer Reference/link-token-contracts.md [hunk 2]
```diff
@@ -124,7 +122,7 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 > 🚰Mumbai Faucet
 >
-> Testnet LINK and MATIC are available from <a href="https://faucet.matic.network/" target="_blank" rel="noreferrer, noopener">the official matic faucet</a> and <a href="https://linkfaucet.protofire.io/mumbai" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
+> Testnet LINK and MATIC are available from <a href="https://faucet.matic.network/" target="_blank" rel="noreferrer, noopener">the official matic faucet</a> and <a href="https://faucets.chain.link/mumbai" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
```
Comment (ainsleybc) [inline docs/Developer Reference/link-token-contracts.md:125]: same here

File: docs/Developer Reference/link-token-contracts.md [hunk 3]
```diff
@@ -186,7 +184,7 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 > 🚰Avax Faucet
 >
-> Community Faucet: https://linkfaucet.protofire.io/fuji
+> Community Faucet: https://faucets.chain.link/fuji
```
Comment (ainsleybc) [inline docs/Developer Reference/link-token-contracts.md:187]: same here

File: docs/Developer Reference/link-token-contracts.md [hunk 4]
```diff
@@ -203,7 +201,7 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 > 🚰Fantom Faucet
 >
-> Community Faucet: https://linkfaucet.protofire.io/fantomtestnet  
+> Community Faucet: https://faucets.chain.link//fantomtestnet
```
Comment (ainsleybc) [inline docs/Developer Reference/link-token-contracts.md:204]: same here

File: docs/Developer Reference/link-token-contracts.md [hunk 5]
```diff
@@ -220,7 +218,7 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 > 🚰Arbitrum Faucet on Rinkeby
 >
-> Community Faucet: https://linkfaucet.protofire.io/rinkebyarbitrum
+> Community Faucet: https://faucets.chain.link/arb-rinkeby
```
Comment (ainsleybc) [inline docs/Developer Reference/link-token-contracts.md:221]: same here

File: docs/Developer Reference/user-guides/acquire-link.md [hunk 6]
```diff
@@ -10,19 +10,21 @@ This page will show you how to obtain and add testnet LINK to MetaMask. If you a
 
 # Testnet LINK Faucet
 
-* Navigate to the Chainlink Kovan Faucet site: <a href="https://kovan.chain.link/" target="_blank" rel="noreferrer, noopener">https://kovan.chain.link</a>
+* Navigate to the Chainlink Kovan Faucet site: <a href="https://faucets.chain.link/kovan" target="_blank" rel="noreferrer, noopener">https://faucets.chain.link/kovan</a>
 
 > 👍 Note
 > 
-> You can also get testnet LINK from: [Rinkeby](https://rinkeby.chain.link/)
+> You can also get testnet LINK from: [Rinkeby](https://faucets.chain.link/rinkeby)
```
Comment (ainsleybc) [inline docs/Developer Reference/user-guides/acquire-link.md:17]: Kind of feel like this shouldn't be here, or we should give the full list of networks you can use

File: docs/Using Randomness/vrf-contracts.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ For implementation details, read [Introduction to Chainlink VRF](../chainlink-vr
 
 > 🚰Mumbai Faucet
 >
-> Testnet LINK and MATIC are available from <a href="https://faucet.matic.network/" target="_blank" rel="noreferrer, noopener">the official matic faucet</a> and <a href="https://linkfaucet.protofire.io/mumbai" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
+> Testnet LINK and MATIC are available from <a href="https://faucet.matic.network/" target="_blank" rel="noreferrer, noopener">the official matic faucet</a> and <a href="https://faucets.chain.link/mumbai" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
```
Comment (ainsleybc) [inline docs/Using Randomness/vrf-contracts.md:37]: same here

File: docs/Using Randomness/vrf-contracts.md [hunk 8]
```diff
@@ -64,7 +64,7 @@ For implementation details, read [Introduction to Chainlink VRF](../chainlink-vr
 
 > 🚰BSC Faucet
 >
-> Testnet LINK is available from <a href="https://linkfaucet.protofire.io/bsctest" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
+> Testnet LINK is available from <a href="https://faucets.chain.link/bsctest" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
```
Comment (ainsleybc) [inline docs/Using Randomness/vrf-contracts.md:67]: last one

**Review by ainsleybc — CHANGES_REQUESTED** [review]
Maybe check with the docs owners before making any of my suggested changes..

File: docs/Developer Reference/link-token-contracts.md [hunk 9]
```diff
@@ -30,8 +30,7 @@ The LINK token is an ERC677 token that inherits functionality from the ERC20 tok
 
 > 🚰Kovan Faucets
 >
-> Testnet LINK is available from https://kovan.chain.link/
-> Testnet ETH is available from https://linkfaucet.protofire.io/kovan
+> Testnet LINK/ETH is available from https://faucets.chain.link/kovan
```
Comment (dwightjl) [inline docs/Developer Reference/link-token-contracts.md:33]:
```suggestion
> Testnet LINK and ETH are available from https://faucets.chain.link/kovan
```

File: docs/Using Randomness/vrf-contracts.md [hunk 10]
```diff
@@ -91,8 +91,7 @@ For implementation details, read [Introduction to Chainlink VRF](../chainlink-vr
 
 > 🚰Kovan Faucets
 >
-> Testnet LINK is available from https://kovan.chain.link/
-> Testnet ETH is available from https://linkfaucet.protofire.io/kovan
+> Testnet LINK/ETH is available from https://faucets.chain.link/kovan
```
Comment (dwightjl) [inline docs/Using Randomness/vrf-contracts.md:94]:
```suggestion
> Testnet LINK and ETH are available from https://faucets.chain.link/kovan
```

File: _includes/samples/DataProviders/AlarmClock.sol [hunk 11]
```diff
@@ -1,8 +1,7 @@
 /** This example code is designed to quickly deploy an example contract using Remix.
  *  If you have never used Remix, try our example walkthrough: https://docs.chain.link/docs/example-walkthrough
  *  You will need testnet ETH and LINK.
- *     - Kovan ETH faucet: https://faucet.kovan.network/
- *     - Kovan LINK faucet: https://kovan.chain.link/
+ *     - Kovan LINK/ETH faucet: https://faucets.chain.link/kovan
```
Comment (dwightjl) [inline _includes/samples/DataProviders/AlarmClock.sol:4]:
```suggestion
 *     - Kovan LINK and ETH faucet: https://faucets.chain.link/kovan
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Review by dwightjl — APPROVED** [review]

**Review by PatrickAlphaC — APPROVED** [review]

**Comment by dwightjl** [comment]
I tested the preview links and confirmed with Bradford that this is ready to merge.

**Pull request merged**